docs/rtk-query/usage/pagination.mdx

```diff
@@ -76,6 +76,43 @@ const PostList = () => {
 };
 ```
 
+## Pagination and Caching
+
+If you have caching enabled for your paginated queries, it maybe a good idea to invalidate the `LIST` cache as well when you do a `delete` mutation.
```

Comment:
```suggestion
It is a common use-case to utilize tag invalidation to perform
[automated re-fetching](./automated-refetching.mdx) with RTK Query.

A potential pitfall when combining this with pagination is that your paginated query may only
provide a _partial_ list at any given time, and hence not `provide` tags for entity IDs that
fall on pages which aren't currently shown. If a specific entity is deleted that falls on an
earlier page, the paginated query will not be providing a tag for that specific ID, and will
not be invalidated to trigger re-fetching data. As a result, items on the current page that
should shift one item up will not have done so, and the total count of items and/or pages
may be incorrect.

A strategy to overcome this is to ensure that the `delete` mutation always `invalidates` the
paginated query, even if the deleted item is not _currently_ provided on that page. We can
leverage the concept of
[advanced invalidation with abstract tag ids](./automated-refetching.mdx#advanced-invalidation-with-abstract-tag-ids)
to do this by `providing` a `'Posts'` tag with the `'PARTIAL-LIST'` ID in our paginated query,
and `invalidating` that corresponding tag for any mutation that should affect it.
```

This adds more detail about the specific scenario we're trying to address, and specifically refers to the tag invalidation usage page to provide context about the topic.

It also describes it using a 'PARTIAL-LIST' tag ID rather than just 'LIST' in order to assist in the distinction.